docker-compose/powpeg/reg1.key

```diff
@@ -0,0 +1 @@
+45c5b07fc1a6f58892615b7c31dca6c96db58c4bbc538a6b8a22999aaa860c32
```

Comment:
Make sense

Reply:
If I remove that the users will need to set up the powpeg, as we are using it for development I can't see any problem.

Reply:
If you prefer I can remove but I will be complicated to generate another one because you need to clone rskJ project to generate this key and set up java in your env